optimize-onnx/src/import.cpp

```diff
@@ -203,6 +203,13 @@ std::unique_ptr<Graph> graphProtoToGraph(const onnx::GraphProto& gp) {
   // inputs.
   std::unordered_map<Node*, std::vector<std::string>> inputs_by_node;
 
+  {
```

Comment:
Maybe add some comments here explaining this is for handling ONNX uses empty string to indicate absent of optional inputs?